config/manager/kustomization.yaml

```diff
@@ -14,5 +14,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/redhat-appstudio/integration-service
-  newTag: next
+  newName: quay.io/jsztuka/integration-service
```

Comment:
Is this expected to use an image from a private  account here?

Reply:
Oh it changes the source image when you run make docker-build docker-push, we want to use redhat-appstudio one.

Reply:
I just double checked and can confirm that this was resolved.